docs/components/movement-sensor/viam-visual-odometry.md

```diff
@@ -0,0 +1,253 @@
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "camera_name": "<your-camera-name>",
+     "time_between_frames_s": <time_seconds>,
+     "lowe_ratio_threshold": <lowe_ratio_threshold>
+   }
+   ```
+
+   Provide the same camera name as you used inn step 3.
+   See the [Attributes](#attributes) section for more information on the other attributes.
+
+1. Click **Save config** at the bottom of the page.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "modules": [
+    {
+      "name": "<your-visual-odometer-name>",
+      "executable_path": "</path/to/run.sh>",
+      "type": "local"
+    }
+  ],
+  "components": [
+    {
+      "name": "<your-camera-name>",
+      "type": "camera",
+      "model": "webcam",
+      "attributes": {
+        "video_path": "<path-to-video-stream>",
+        "height_px": <height>,
+        "width_px": <width>,
+        "intrinsic_parameters": {
+          "ppx": <ppx>,
+          "ppy": <ppy>,
+          "fx": <fx>,
+          "fy": <fy>
+        },
+        "distortion_parameters": {
+          "rk3": <rk3>,
+          "tp1": <tp1>,
+          "tp2": <tp2>,
+          "rk1": <rk1>,
+          "rk2": <rk2>
+        }
+      },
+      "depends_on": []
+    },
+    {
+      "name": "<your_movement_sensor_name>",
+      "type": "movement_sensor",    
+      "namespace": "rdk",
+      "model": "viam:visual_odometry:opencv_orb",
+      "attributes": {
+        "camera_name": "<your-camera-name>", 
+        "time_between_frames_s": <time_seconds>, 
+        "lowe_ratio_threshold": <lowe_ratio_threshold>
+      },
+      "depends_on": [
+        "<your-camera-name>"
+      ]
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "modules": [
+    {
+      "name": "my-odometry-module",
+      "executable_path": "/path/to/run.sh",
+      "type": "local"
+    }
+  ],
+  "components": [
+    {
+      "name": "my-camera",
+      "type": "camera",
+      "model": "webcam",
+      "attributes": {
+        "video_path": "FDF90FEB-59E5-4FCF-AABD-DA03C4E19BFB",
+        "height_px": 720,
+        "width_px": 1280,
+        "intrinsic_parameters": {
+          "ppx": 446,
+          "ppy": 585,
+          "fx": 1055,
+          "fy": 1209
+        },
+        "distortion_parameters": {
+          "rk3": -0.03443,
+          "tp1": 0.01364798,
+          "tp2": -0.0107569,
+          "rk1": -0.1621,
+          "rk2": 0.13632
+        }
+      },
+      "depends_on": []
+    },
+    {
+      "name": "my_movement_sensor",
+      "type": "movement_sensor",
+      "namespace": "rdk",
+      "model": "viam:visual_odometry:opencv_orb",
+      "attributes": {
+        "camera_name": "my-camera", 
+        "time_between_frames_s": 0.2, 
+        "lowe_ratio_threshold": 0.75
+      },
+      "depends_on": [
+        "my-camera"

```

Comment:
It should work with the `depends_on` field  empty. An error will be logged if the attribute `camera_name` is empty.